src/ManualClient.py

```diff
@@ -394,8 +407,9 @@ def update_tracker_and_locations_table(self, update_highlights=False):
                                         # Get the item name from the item Label, minus quantity, then do a lookup for count
                                         old_item_text = item.text
                                         item_name = re.sub("\s\(\d+\)$", "", item.text)
+                                        item_id = self.ctx.item_names_to_id[item_name]
                                         item_data = self.ctx.get_item_by_name(item_name)
```

Comment:
item_data looks to not be used anymore after adding the item_id var.

Reply:
Ah, yep. My IDE didn't catch it because we reuse the name later in the method.  Fixed.